web/template/lpa_details_saved.gohtml

```diff
@@ -36,7 +36,7 @@
             {{ if .Lpa.Tasks.PayForLpa.IsCompleted }}
                 <a href="{{ link .App (.App.Paths.Dashboard.String) }}" class="govuk-button">{{ tr .App "returnToDashboard" }}</a>
             {{ else }}
-                {{ if eq "paper" .Lpa.CertificateProvider.CarryOutBy }}
+                {{ if and (eq "paper" .Lpa.CertificateProvider.CarryOutBy) .IsFirstCheck }}
```

Comment:
can't believe I missed something off when I did the enum stuff

Reply:
Doh, I was on autopilot. I'll bundle up with the next ticket too 👍